test/integration/util/util.go

```diff
@@ -120,7 +122,7 @@ type OcRunner struct {
 }
 
 func NewOcRunner() *OcRunner {
-	p, _ := exec.LookPath("oc")
+	p := path.Join(os.Getenv(constants.MiniShiftHomeEnv), "cache", "oc", "v1.4.1", "oc")
```

Comment:
I don't get it. Why are you not using '~/.minishift/machines/minishift.json'? That's why we introduced it, right? The LookupPath was a crutch until we could do it. 

So as a minimum the oc path should be read from the config file. 

There are a couple of other issues here which need addressing. Preferably sooner than later. 

* The integration tests are interfering atm with the default Minishift directory. As part of the integration test setup we should create a known directory within our source tree (e.g. out/integration-test or something). Easiest would be to create the directory in make and then set this directory as value for MINISHIFT_HOME when calling the integration test. Problem is that in this case one needs to drive the test via Make. 
* The second problem is constants.MachineName, we need to take this value from an env variable as well, otherwise even though we can separate the home directories, we still cannot create multiple machines since the names are clashing. It needs to be tested whether the name can easily change. There might be some hard coded values which will cause trouble.

Bottom line, as a bare minimum we need to adjust the path. I'd love to see the other two issues addressed as well, but let's discuss.